src/app/+browse-by/+browse-by-metadata-page/browse-by-metadata-page.component.ts

```diff
@@ -48,7 +50,7 @@ export class BrowseByMetadataPageComponent implements OnInit {
    * The pagination config used to display the values
    */
   paginationConfig: PaginationComponentOptions = Object.assign(new PaginationComponentOptions(), {
-    id: 'browse-by-metadata-pagination',
+    id: 'bbm',
```

Comment:
This does look like an instance where the id might be at fault. We'll look into it

Reply:
@artlowel ; Just a sidenote, I just realized there's also a separate sorting issue reported in #1071.  I don't believe that ticket is the same as the bug reported here, but I could be mistaken. So, it may be worth glancing at whether these issues are related at all.

Reply:
I noticed that a similar call is made on demo dspace when reproducing the issue there. I also checked the pagination code again and did not see how the pagination would necessarily interfere with this. 
I did fix an issue with the sorting direction in the dropdown not updating.